common/compositor/va/varenderer.cpp

```diff
@@ -357,4 +296,39 @@ int VARenderer::DrmFormatToRTFormat(int format) {
   return 0;
 }
 
+bool VARenderer::CreateContext() {
+  DestroyContext();
+
```

Comment:
Can we do something similar to  this: ScopedVASurfaceID surface_in(va_display_);

vasurface can allocate this. We anyway have tripple buffer support and recycle them, so we should easily avoid re-creating va surfaces.

Reply:
vaCreateSurface takes very limited time so we don't need to make thing complicated.

Reply:
We already have surface management in place, nothing new which needs to be done specifically for this. vasurface in our side needs to allocate and de-allocate surface appropriately. Nothing else should be needed.